app/src/ui/preferences/git.tsx

```diff
@@ -151,7 +165,14 @@ export class Git extends React.Component<IGitProps, IGitState> {
         )}
 
         <p className="git-settings-description">
-          These preferences will edit your global Git config.
+          These preferences will{' '}
+          {this.props.selectedExternalEditor && this.state.globalGitConfigPath
+            ? (
+                <LinkButton onClick={this.openGlobalGitConfigInEditor}>
+                  edit your global Git config file
+                </LinkButton>
+              ) + '.'
+            : 'edit your global Git config file.'}
         </p>
```

Comment:
```suggestion
              )
            : 'edit your global Git config file.'}
            .
        </p>
```
Also, I pulled this down to make sure I was sending you in the right direction, and noticed that your link is not rendering. It is because you cannot use that `+` to concatenate here. But, you can just put a period after your `{}` since both outcomes require it and it is just plain text.